test/performance/benchmarks/broker-gcp/continuous/200-broker-gcp.yaml

```diff
@@ -0,0 +1,131 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: broker-gcp-receiver
+  namespace: default
+  labels:
+    role: broker-gcp-receiver
+spec:
+  schedule: "0/15 * * * *"
+  # History must be zero to ensure no failed pods stick around and block the next job
+  successfulJobsHistoryLimit: 0
+  failedJobsHistoryLimit: 0
+  jobTemplate:
+    spec:
+      completions: 1
+      parallelism: 1
+      backoffLimit: 0
+      # Allow up to 14 minutes, then clean up to make room for the next attempt
+      activeDeadlineSeconds: 840
+      template:
+        metadata:
+          labels:
+            role: broker-gcp-receiver
+        spec:
+          serviceAccountName: perf-gcpbroker
+          restartPolicy: Never
+          containers:
+          - name: sender-receiver
+            image: ko://knative.dev/eventing/test/test_images/performance
+            args:
+              - "--roles=sender,receiver"
+              - "--sink=http://broker-ingress.cloud-run-events/default/gcp"
+              - "--aggregator=broker-gcp-aggregator:10000"
+              - "--pace=100:10,200:20,400:30,500:60,600:60"
+            env:
+              - name: POD_NAME
+                valueFrom:
+                  fieldRef:
+                    fieldPath: metadata.name
+              - name: POD_NAMESPACE
+                valueFrom:
+                  fieldRef:
+                    fieldPath: metadata.namespace
+            resources:
+              requests:
+                cpu: 1000m
+                memory: 2Gi
+            ports:
+              - name: cloudevents
+                containerPort: 8080
+
+---
+
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: broker-gcp-aggregator
+  namespace: default
+  labels:
+    role: broker-gcp-aggregator
+spec:
+  schedule: "0/15 * * * *"
+  # History must be zero to ensure no failed pods stick around and block the next job
+  successfulJobsHistoryLimit: 0
+  failedJobsHistoryLimit: 0
+  jobTemplate:
+    spec:
+      completions: 1
+      parallelism: 1
+      backoffLimit: 0
+      # Allow up to 14 minutes, then clean up to make room for the next attempt
+      activeDeadlineSeconds: 840
+      template:
+        metadata:
+          labels:
+            role: broker-gcp-aggregator
+        spec:
+          serviceAccountName: perf-gcpbroker
+          restartPolicy: Never
+          containers:
+          - name: aggregator
+            image: ko://knative.dev/eventing/test/test_images/performance
+            args:
+              - "--roles=aggregator"
+              # set to the number of sender + receiver (same image that does both counts 2)
+              - "--expect-records=2"
+              - "--mako-tags=channel=pubsub"
```

Comment:
Since this broker doesn't use a channel, it shouldn't add the `channel=pubsub` tag. Maybe use a `broker.class=googlecloud` tag. But it's academic because this benchmark will only ever be used to test a single broker class anyway.